cdk/tooling_environment/stacks/grafana/lambda_functions_dashboard.template.json

```diff
@@ -225,39 +225,7 @@
         "label": "Measure name",
         "multi": true,
         "name": "measure",
-        "options": [
-          {
-            "selected": true,
-            "text": "All",
-            "value": "$__all"
-          },
-          {
-            "selected": false,
-            "text": "execution",
-            "value": "execution"
-          },
-          {
-            "selected": false,
-            "text": "duration_ms",
-            "value": "duration_ms"
-          },
-          {
-            "selected": false,
-            "text": "max_memory_used_mb",
-            "value": "max_memory_used_mb"
-          },
-          {
-            "selected": false,
-            "text": "billed_duration_ms",
-            "value": "billed_duration_ms"
-          },
-          {
-            "selected": false,
-            "text": "memory_size_mb",
-            "value": "memory_size_mb"
-          }
-        ],
-        "query": "execution,duration_ms,max_memory_used_mb,billed_duration_ms,memory_size_mb",
+        "query": "attempt,duration_ms,max_memory_used_mb,billed_duration_ms,memory_size_mb",
```

Comment:
Looks very good!
I'd also like to discuss if we want to add something like "executions_succeeded", "_failed" meaning, eventually, after retries